sshnp_docker_demo/db-template.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#
+# DO NOT RUN THIS SCRIPT. THIS IS MEANT TO BE RUN BY ANOTHER SCRIPT IN ONE OF THE THREE DIRECTORIES
+#
+
+if [ -z "$1" ]
+then
+    echo "Usage: ./db.sh <sshnp/sshnpd/sshrvd>"
+    exit 1
+fi
+
+rm Dockerfile
+cp ../Dockerfile .
+docker container prune -f
```

Comment:
All of these `docker` commands assume that it can be run without `sudo`, which isn't a safe assumption

Reply:
Would I just `sudo docker` ?

https://github.com/atsign-foundation/at_demos/pull/97/commits/1aaad8c04a64f11085e9a2619e7c1883c5789d4d